ts/IterableInboxMessageList.tsx

```diff
@@ -39,11 +39,11 @@ const IterableInboxMessageList = ({
                index={index}
                last={last}
                rowViewModel={rowViewModel}
-               messageListItemLayout={messageListItemLayout}
                customizations={customizations}
-               //swipingCheck={(swiping : boolean) => setSwiping(swiping)}
+               messageListItemLayout={messageListItemLayout}
                deleteRow={(messageId: string) => deleteRow(messageId)}
                handleMessageSelect={(messageId: string, index: number) => handleMessageSelect(messageId, index)}
+               //swipingCheck={(swiping : boolean) => setSwiping(swiping)}
```

Comment:
Same comment about relevance for this line too.

Reply:
Deleted as well.